common/string_helpers.cpp

```diff
@@ -14,7 +14,7 @@
 
 namespace Carbon {
 
-static constexpr llvm::StringRef TripleQuotes = R"(""")";
+static constexpr llvm::StringRef TripleQuotes = R"(''')";
```

Comment:
```suggestion
static constexpr llvm::StringRef TripleQuotes = "'''";
```

No longer needs the R